steps/run/test-metadata.yaml

```diff
@@ -0,0 +1,15 @@
+# connections:
```

Comment:
I don't think we necessarily want to include these files here, at least until we figure out what a testing framework might look like.